scripts/functions.sh

```diff
@@ -36,3 +36,14 @@ export RPCD_DIR="${BASE_DIR}/rpcd"
 function run_ansible {
   openstack-ansible ${ANSIBLE_PARAMETERS} --forks ${FORKS} $@
 }
+
+function add_config {
+  src="$1"
+  src_path="${BASE_DIR}/scripts/config/${src}.yml"
+  if [[ -e "${src_path}" ]]; then
+      echo "Adding vars from $src_path to $RPCD_VARS: $(cat $src_path)"
+      cat ${src_path} >> $RPCD_VARS
```

Comment:
Is there a risk to duplicate some vars, or mess up with user configured files?
If this is only used for gating, that shouldn't be a problem.
However, it looks like ``add_config ceph`` is not only used in gating.
So all of the variables should probably be copied to a separate file (if there is at some point an idea to use this for non gating).